kzg_prover/src/circuits/tests.rs

```diff
@@ -59,25 +60,41 @@ mod test {
         // (The first column is the user IDs)
         let balance_column_range = 1..N_CURRENCIES + 1;
 
+        // The Custodian communicates the polynomial length to the Verifier
+        let poly_length = u64::try_from(advice_polys.advice_polys[0].len()).unwrap();
```

Comment:
Can we retrieve `poly_length` from 2^K instead of this? There shouldn't be any substantial difference but using K reflects better what we are gonna do in production, namely fetching the K from the verifier smart contract.

(This also applies to `kzg.rs` in `benches`

Reply:
Done